c/src/ml-api-service-db.cc

```diff
@@ -0,0 +1,268 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/**
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * @file ml-api-service-db.c
+ * @date 07 Mar 2022
+ * @brief NNStreamer/Service C-API implementation
```

Comment:
Please check the descriptions of the doxgen tag.

Reply:
Updated! Thanks.